### PR TITLE
MUMMNG-1812 : Payroll Widget Redesign

### PR DESCRIPTION
**Pull request by Daha5**
An update on the payroll widget design.

Before:

before

After:

after

The after screenshot was taken on predev while the before was taken on test, which is why the earnings statements don't load on after screenshot (Unless that's not the case and I unknowingly broke something). However all that was changed for the top part was the removal of the greek temple icon as well as the text from {date} Statement to {date} Earnings Statement.

### CODE REVIEW

**Comment by apetro** [comment]
:+1:. 

Appreciate your updating the `mock` data, I do think that's important for keeping the development/mock/demo environment realistic and productive.

Of course, need to update `uPortal` repo to queue up the entity change for migration as well.

**Comment by timlevett** [comment]
^ same :+1:

**Comment by Daha5** [comment]
Ah ok, will do!

**Comment by Daha5** [comment]
Hold off on merging this, I think I broke something

**Comment by Daha5** [comment]
Ok this should be all good. 
Final screenshot.

final payroll

File: angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc [hunk 1]
```diff
@@ -69,7 +69,7 @@
...
...ata-empty='isEmpty'></loading-gif>\n <ul class='widget-list'><li ng-repeat='item in content.report |orderBy: [\"-paid.substring(6)\",\"-paid.substring(0,2)\",\"-paid.substring(3,5)\"] | limitTo:3' class='center'><a href='/portal/p/earnings-statement/max/earning_statement.pdf.resource.uP?pP_docId={{item.docId}}' target='_blank'> {{item.paid}}Earnings Statement</a></li></ul>\n <div ng-if='isEmpty' style='padding: 10px; font-size: 14px;'><i class='fa fa-exclamation-triangle fa-3x pull-left' style='color: #b70101;'></i><span style='color: #898989;'>We had a problem finding your statements (or you don't have any).</span></div>\n <div style='background-color: #EAEAEA; border-radius:4px;padding:10px; margin-top:10px;'>\n <span class='bold display-block left' style='text-align: left; padding-left: 10px; font-size: 14px;'>Launch payroll to find::</span> \n <ul style='text-align: left;list-style-type: disc; font-size: 12px;'>\n <li>Earnings statements</li>\n <li>Tax statements (W-2, 1095-C, etc.)</li>\n <li>W-4 and direct deposit forms</li>\n </ul>\n </div>\n </div>\n <a class='btn btn-default launch-app-button' href='/portal/p/earnings-statement'>See all payroll information</a>\n         ",
```
Comment (apetro) [inline angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc]: `{{item.paid}}Earnings Statement</a>`

I think our style guidance would have this camel case, as in

`{{item.paid}}Earnings statement</a>`

Do I have that style guidance right?

Comment (timlevett) [inline angularjs-portal-mock-portal/src/main/webapp/web/layoutDoc]: Looks like it was changed to just `<date> Statement`.

**Comment by timlevett** [comment]
👍

**Comment by timlevett** [comment]
@Daha5 make sure to update the entity files so we can see these changes in actual environments.

Thanks,

@timlevett

**Pull request merged**